cluster/images/kubemark/Makefile

```diff
@@ -15,8 +15,19 @@
 # build Kubemark image from currently built binaries containing both 'real' master and Hollow Node.
 # This makefile assumes that the kubemark binary is present in this directory.
 
+REGISTRY?="gcr.io"
+PROJECT?="google_containers"
+
 all:
-	docker build --pull -t gcr.io/$(PROJECT)/kubemark .
-	gcloud docker -- push gcr.io/$(PROJECT)/kubemark
+	gcloudpush
+
+build:
+	docker build --pull -t $(REGISTRY)/$(PROJECT)/kubemark .
+
+gcloudpush: build
```

Comment:
Instead of having one such push rule per provider, how about we use a DOCKER_PUSH_CMD variable (which defaults to `gcloud docker -- push`). We'd then just need only the push rule.